src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -142,7 +139,9 @@ class CheckBoxComponent extends React.Component<CheckBoxProps> {
         width: iconWidth,
         height: iconHeight,
         borderRadius: iconBorderRadius,
-        backgroundColor: iconTintColor,
+        tintColor: iconTintColor,
+        // FIXES https://github.com/necolas/react-native-web/issues/1319
+        opacity: this.props.checked ? 1.0 : 0.0,
```

Comment:
:( Given that the correctness of the operation of the library on the web is not the main priority, I suggest not generalizing the solution to this issue for all platforms. Could it be done somehow:
```
Platform.select({
  ios: 1.0,
  android: 1.0,
  default: this.props.checked ? 1.0 : 0.0,
})
```
?

Reply:
I guess this would be kinda useless condition to achieve the same behavior. This could have the sense to do this in `.web` components, but it requires exporting additional style props in mobile components

Reply:
Anyway I believe we need another one solution